core/src/main/python/wlsdeploy/tool/create/domain_typedef.py

```diff
@@ -109,6 +109,13 @@ def domain_type_has_jrf_resources(self):
         """
         return 'JRF-MAN-SVR' in self.get_server_groups_to_target()
 
+    def domain_type_is_restricted_jrf(self):
+        """
+        A workaround for a bug with applyJRF in online with RestrictedJRF
+        :return:
+        """
+        return self.get_domain_type() == 'RestrictedJRF'
+
```

Comment:
CER and reviewers - do you think there is a case where a user might create their own typedef based on RestrctedJRF?  We could add a flag instead of checking the name directly, but not sure if it is worth it.
I noticed there is another direct name check in this file for JRF, but it is not used any more.

Reply:
This is a good point. I put it in as a temporary until come up with a replacement. But once in the code, it probably will not come out. I do not know how many upper stack products use RestrictedJRF instead of JRF when installing their own product templates. I don't think we can use the server groups for RestrictedJRF, but we could look at the templates. This is a bit of code because early version have the physical name of the template jar and later versions, have the template name (from the template-info.xml)